web/src/components/Upload/UploadValidationResultsTable/UploadValidationResultsTable.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render } from '@redwoodjs/testing/web'
+
+import UploadValidationResultsTable from './UploadValidationResultsTable'
+import { Severity } from './UploadValidationResultsTable'
+
+describe('UploadValidationResultsTable', () => {
+  it('renders successfully when empty', () => {
+    expect(() => {
+      render(<UploadValidationResultsTable errors={[]} />)
+    }).not.toThrow()
+  })
+  it('renders successfully when given minimal results', () => {
```

Comment:
Would it be possible to add a test case ensuring that the error is displayed as well?

Reply:
Done! (assuming you mean an Error-type row)